tests/actionsFactory.spec.ts

```diff
@@ -129,33 +139,34 @@ describe("actionsFactory test", () => {
 
         it("fails without uri", () => {
             const actionJson = {
-                [ActionJsonKey.ActionType]: ActionType.File
+                actionType: "FILE"
             };
             uriMock.expects("parse").withExactArgs('');
             uriMock.expects("parse").withExactArgs(undefined, true).throws(new Error('Failed!'));
             expect(() => ActionsFactory.createAction(actionJson)).
-                to.throw(`Failed to parse field ${ActionJsonKey.Uri}: undefined for actionType=${ActionType.File}: Failed!`);
+                to.throw(`Failed to parse field uri: undefined for actionType = FILE: Failed!`);
         });
     });
 
     describe("create action fails", () => {
         it("when no action type defined", () => {
             const actionJson = {};
-            expect(() => ActionsFactory.createAction(actionJson)).to.throw(`${ActionJsonKey.ActionType} is missing`);
+            expect(() => ActionsFactory.createAction(actionJson)).to.throw(`actionType is missing`);
         });
 
         it("when no unsupported action type used", () => {
             const actionJson = {
-                [ActionJsonKey.ActionType]: "Unsupported"
+                actionType: "Unsupported"
             };
-            expect(() => ActionsFactory.createAction(actionJson)).to.throw(`Action with ${ActionJsonKey.ActionType}=Unsupported could not be created from json file`);
+            expect(() => ActionsFactory.createAction(actionJson)).to.throw(`Action with actionType = Unsupported could not be created from json file`);
```

Comment:
I think the value of the actionType in the error message should be quoted for increased readability.